src/context.ts

```diff
@@ -59,6 +61,8 @@ export function getInputs(): Inputs {
     file: core.getInput('file'),
     githubToken: core.getInput('github-token'),
     labels: core.getMultilineInput('labels'),
+    lint: core.getBooleanInput('lint'),
+    lintFailOn: core.getInput('lint-fail-on') || 'error',
```

Comment:
Do I need to have this default error string?

Reply:
You do not need this default, no (with no value, the flag just won't be provided)